routes/routes.js

```diff
@@ -6,6 +6,17 @@ router.post('/factorial', (req, res) => {
 
 router.post('/palindrome', (req, res) => {
     //Palindrome logic goes here
+    function palindrome(str) {
+ var re = /[^A-Za-z0-9]/g;
+ str = str.toLowerCase().replace(re, '');
+ var len = str.length;
+ for (var i = 0; i < len/2; i++) {
+   if (str[i] !== str[len - 1 - i]) {
+       return false;
+   }
+ }
+ return true;
+}
 })
```

Comment:
Here you have simply added a function directly. What is required to be done here is take take the str variable from the body of the request and apply the logic you have made, after which a response back as well.

Reply:
ok i will do it asap